plugin/trino-postgresql/src/main/java/io/trino/plugin/postgresql/PostgreSqlConfig.java

```diff
@@ -55,4 +56,16 @@ public PostgreSqlConfig setIncludeSystemTables(boolean includeSystemTables)
         this.includeSystemTables = includeSystemTables;
         return this;
     }
+
+    public boolean isStringPushdownWithCollate()
+    {
+        return enableStringPushdownWithCollate;
+    }
+
+    @Config("postgresql.experimental.string-pushdown-with-collate")
```

Comment:
```suggestion
    @Config("postgresql.experimental.string-pushdown-with-collate.enabled")
```

since that seems to be convention for JdbcMetadataConfig and other recent configs introduced.

Reply:
Adjust the getter and setter names accordingly.

Reply:
Revised to `postgresql.experimental.enable-string-pushdown-with-collate`